index.html

```diff
@@ -56,8 +56,8 @@ <h1>How to Submit an Erratum?</h1>
 
     <main>
       <!-- The data-erratalabel should include one label that filters the errata -->
-      <section data-erratalabel="@@@@">
-        <h1 class="todo">Open Errata on the “@@@@” Recommendation</h1>
+      <section data-erratalabel="@@@@1">
+        <h1 class="todo">Open Errata on the “@@@@1” Recommendation</h1>
```

Comment:
is there actually a difference between an open errata and a close errata?

Reply:
If an errata issue is close, it won't be displayed (at least that is the intention:-)